### DISCUSSION TITLE
Mentioning `noValidate` on Input docs

### DISCUSSION THREAD

**Discussion by GiantRobots** [twilio-labs/paste#1166] [Ideas] [answered]
When rendering an input box with `<input required />` (or other html5 validation API's) the default browser behavior will take place (Which looks like this)

![Screen Shot 2021-02-04 at 5 09 35 PM](https://user-images.githubusercontent.com/1903839/106971262-c73e9280-670b-11eb-886b-5ec185cfc7a5.png)

We have design system errors that we'd like to use that look like this.

![Screen Shot 2021-02-04 at 5 10 02 PM](https://user-images.githubusercontent.com/1903839/106971448-31efce00-670c-11eb-8214-bdba68c2ad52.png)

`noValidate` is the html5 way to turn off form validation and I don't see it recommended in the docs. Since we have other accessibility information we probably want to advise developers to turn it off in favor of our own error states.

**Comment by kate-porter** [comment, marked as answer]
Hey @GiantRobots - thanks so much for this suggestion! Great call to add it to the docs. I've created a ticket for it in our backlog: https://issues.corp.twilio.com/browse/DSYS-2614
